### ISSUE TITLE
Sigmoid attention?

### ISSUE THREAD

**Issue by lhallee** [pytorch-labs/attention-gym#116]
Hello,

Is there any planned support for different activation functions, like replacing the softmax with sigmoid?
Seem promising https://arxiv.org/pdf/2502.00281
Best,
Logan

**Comment by drisspg** [comment]
[PR #195](https://github.com/meta-pytorch/attention-gym/pull/195)) :)
